### DISCUSSION TITLE
Add a new prop on the Devtools that would set it's placement (top-right, bottom-left, etc) + Chrome extension

### DISCUSSION THREAD

**Discussion by arjunvegda** [jotaijs/jotai-devtools#86] [Ideas]
From: @maylortaylor in https://github.com/jotaijs/jotai-devtools/issues/82
> One question though, is there a way move the Jotai Devtools icon or is there a plan to make a chrome extension in the future? The icon is covering some of my UI that I need.
>    
> I would also suggest adding a new prop on the Devtools that would set it's placement (top-right, bottom-left, etc)

**Comment by arjunvegda** [comment]
> is there a way move the Jotai Devtools icon

Yes, there is a way you could style it on your end. We expose static class names that lets you tweak the styles of certain elements in Jotai DevTools.

Here is a [CodeSandbox](https://codesandbox.io/s/jotai-devtools-demo-discussion-86-yqzxy8?file=/src/styles.css:54-118)

```css
.jotai-devtools-trigger-button {
  left: unset;
  right: 10px;
}
```

> I would also suggest adding a new prop on the Devtools that would set it's placement (top-right, bottom-left, etc)

This is on my to-do list, but I don't have enough time to execute this. We're accepting PRs if you're interested! My goal is to make it draggable to any corner in the viewport + accept a prop trigger placement prop.

> or is there a plan to make a chrome extension in the future?

There is an ongoing discussion around this in https://github.com/jotaijs/jotai-devtools/discussions/59#discussioncomment-6433090. I posted a small update today 😁

**Comment by TinsFox** [comment]
Is it possible to consider the built-in drag effect, so that users can drag icons arbitrarily.
